tests/Model/ValueObject/String/Behaviour/MockHasMaxLengthTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Model\ValueObject\String\Behaviour;
+
+use CultuurNet\UDB3\Http\ApiProblem\ApiProblem;
+use PHPUnit\Framework\TestCase;
+
+class MockHasMaxLengthTest extends TestCase
+{
+    /**
+     * @dataProvider dataProvider
+     */
+    public function testMaxLengthValidation(string $value, int $maxLength, bool $expectException): void
+    {
+        if ($expectException) {
+            $this->expectException(ApiProblem::class);
+        }
+
+        $mock = new MockHasMaxLength($value, $maxLength);
+
+        // Check for the happy path
+        $this->assertTrue($mock->isSuccess());
```

Comment:
This is a side-effect that is only available on the Mock and not in a real scenario were the trait would be used.

Reply:
I know, but I did not know how else to check the happy patchs, because they don't return anything. I could make hasMaxLength() always make return a bool if you prefer? Or if you have another solution?

Reply:
Assert the value from the string, like `it_should_throw_an_exception_if_an_empty_string_is_given`